zk.js/src/utxo.ts

```diff
@@ -358,14 +360,20 @@ export class Utxo {
           "appData length exceeds 16",
         );
       }
-      this.appDataHash = new BN(
-        leInt2Buff(
-          unstringifyBigInts(poseidon.F.toString(poseidon(hashArray))),
-          32,
-        ),
-        undefined,
-        "le",
-      );
+      if (featureFlags.wasmPoseidon) {
+        this.appDataHash = new BN(leInt2Buff(unstringifyBigInts(wasmPoseidon(hashArray).toString()), 32), undefined, "le");
```

Comment:
```suggestion
        this.appDataHash = new BN(wasmPoseidon(hashArray), undefined, "be");
```
Is wasmPoseidon output le or be?

Reply:
Good catch, thanks!